pyproject.toml

```diff
@@ -54,8 +54,8 @@ dev-dependencies = [
 ]
 override-dependencies = [
     "bitsandbytes; sys_platform == 'linux'",
-    "unsloth==2025.3.19 ; sys_platform == 'linux'",
-    "unsloth-zoo==2025.3.17 ; sys_platform == 'linux'",
+    "unsloth==2025.5.1 ; sys_platform == 'linux'",
```

Comment:
Can we remove these from override-dependencies and just keep them in dependencies now?